app/services/api_course_search_service.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class APICourseSearchService < CourseSearchService
+  def default_course_order(outer_scope)
+    outer_scope.order('course.id': :asc)
```

Comment:
I added course id ASC but let me know if you prefer any other way on the default order on the API. Happy to change.

Reply:
One day, it would be nice to stop using consecutive numerical ids, so perhaps it would be better to order by `:created_at` or `:name`

Reply:
Would `updated_at` be the best ordering?

Reply:
Good suggestion. In related to your suggestion I prefer the created_at. I will add the changes.

Reply:
Done!

Reply:
> Would updated_at be the best ordering?
`updated_at DESC` I'd say yes. @dcyoung-dev wdyt?

Reply:
I'm happy with `id` or `created_at` `asc` both of these would preserve the pages in which they appear.
i.e. if more Courses are added they will be on the last page, all other Courses stay on the page they were on. 
If we order by a value that updates then the results on each page get mixed up again.

Reply:
Ok, the `created at asc` is implemented so now all is good. Thank you all 🙏